### ISSUE TITLE
Can the IPT provide an error message when required updates of extensions or cores are the reason for a failed publication process? 

### ISSUE THREAD

**Issue by CecSve** [gbif/ipt#1924]
A common reason for not being able to update/republish a dataset in the IPT is that extensions (and maybe cores?) should be updated. I can see that we alert of the required updates in the [administrations menu](https://ipt.gbif.org/manual/en/ipt/latest/administration#configure-core-types-and-extensions) but could it be possible to get an error message when this is the reason for the publishing process gone wrong?

**Comment by dbloom** [comment]
@CecSve Perhaps rather than an error message it would be more effective, and proactive, to provide IPT Admins with a notification that an update is available for a given core or extension. I make it a regular practice to check the admin menu to see if I need to update, but it could be more helpful receive an message at the top of the IPT (like the yellow/green notices that appear when a mapping is incorrect or some element of the metadata is incomplete) that an update is available or some other indication in the U/I (a red dot on the Admin tab?). I admit I do not always catch or check for update information via GitHub.

**Comment by CecSve** [comment]
Good point @dbloom! It would make more sense to be notified beforehand so you do not go through the whole mapping process before you realize the problem. @mike-podolskiy90 you think it would be possible to have a notification instead of an error message?

**Comment by mike-podolskiy90** [comment]
Yes, of course. I'll see what format it's best to do

**Comment by sformel-usgs** [comment]
Related to this, on the page `myipt/admin/extensions.do` I suggest moving the section, "Synchronize Extensions and Vocabularies" above the "Extensions" section.  It gets buried under the extensions and the first time I encountered it was after I had individually updated all the extensions and then scrolled down.  Not a big deal, just a small Ux thing.

**Comment by mike-podolskiy90** [comment]
@sformel-usgs Good point! I'll include that to the next release

**Comment by mike-podolskiy90** [comment]
This warning is displayed even if the extensions are updated.
